### ISSUE TITLE
Compatibility with TSM

### ISSUE THREAD

**Issue by KevinTyrrell** [neotron/WoW-BulkMail#11]
TSM's custom mail frame bars access to the 'Send Mail' option for BulkMail. Every single time users want to use BulkMail, they must click 'Wow UI' on TSM's mail frame, go to the send mail tab, click BulkMail's send  mail button, then click back into TSM's mail mode. Attaching the ItemSendQueue window onto the TSM frame seems to be a decent solution that may not require too much hassle.

Alternatively, if #10 was to be implemented, the need for this compatibility would not be needed as you could just rely on the auto  bulkmail send setting rather than constantly switching back and forth from TSM's mail frame and the default mail frame.

**Comment by neotron** [comment]
I agree, this is annoying (I use TSM too). However I don't see a way to get the TSM mail frame - it's all private API's and the only UI you can even know if it's toggling is the crafting UI (and even I don't know if even that gives you the frame).

It IS possible to iterate through frames and actually find it though - a bit hacky but might be sufficient.

**Comment by neotron** [comment]
Fixed.
